extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui-templates/build-time/index.html

```diff
@@ -5,7 +5,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="shortcut icon" type="image/png" href="{contextRoot}favicon.ico">
 
-        <script async src="{nonApplicationRoot}_static/es-module-shims/dist/es-module-shims.js"></script>
+        <script async src="{nonApplicationRoot}_static/es-module-shims/1.8.0/dist/es-module-shims.js"></script>
```

Comment:
Isn't this extremely brittle? What if it is updated?

Reply:
This is a bit of a corner case, as this is the thing that make importmaps work when the browser does not support importmaps, so I can not use the importmap to reference this. I can probably look into finding the version from the pom and pass that along in something like `{es-modele-version}`. Let me investigate.

Reply:
This is done